controllers/kserve_inferenceservice_controller.go

```diff
@@ -468,6 +470,14 @@ func (r *OpenshiftInferenceServiceReconciler) createOrUpdateMetricsServiceMonito
 	return nil
 }
 
+func (r *OpenshiftInferenceServiceReconciler) OnDeletionOfKserveInferenceService(ctx context.Context, inferenceService *kservev1beta1.InferenceService) error {
+	log := r.Log.WithValues("InferenceService", inferenceService.Name, "namespace", inferenceService.Namespace)
+
+	log.Info("Deleting Kserve inference service generic route")
```

Comment:
nitpick: Same suggestion in this file about raising log-level for messages that look more like Debug messages.

Reply:
done